AirTote/TabMainPage.xaml.cs

```diff
@@ -0,0 +1,10 @@
+namespace AirTote.Pages;
```

Comment:
名前空間は、なるべくディレクトリ構成とあわせてください。

```suggestion
namespace AirTote;
```